lib/CLI/Parser.hs

```diff
@@ -568,6 +546,39 @@ parseFieldNameWhile whileCond = do
 parseFieldContentsEof :: MParser FieldContents
 parseFieldContentsEof = FieldContents . T.pack <$> P.manyTill P.anySingle P.eof
 
+parseSort :: MParser (Sort, Direction)
+parseSort = do
+  sort <- parseSortMeans
+  void $ P.char ':'
+  direction <- parseSortDirection
+  return (sort, direction)
+
+
+parseSortDirection :: MParser Direction
+parseSortDirection =
+  try (P.string "asc" $> Asc)
```

Comment:
```suggestion
  P.string "asc" $> Asc
```
I predict that this `try` is not needed. Looks like we either need `try` before both alternatives or we don't need them at all.

Reply:
Yeah I don't think it's needed either.